includes/core/server/class-request.php

```diff
@@ -84,6 +85,60 @@ abstract class Request {
 	 */
 	protected $id;
 
+	/**
+	 * Creates a new request, loading dependencies in there.
+	 *
+	 * @param mixed $id The identifier for various update/get/delete requests.
+	 *
+	 * @indexof $this->>routeList
+	 *
+	 * @return static
+	 */
+
+	/**
+	 * Route list.
+	 *
+	 * @var string[]
+	 */
+	private $route_list = [
+		WC_Payments_API_Client::ACCOUNTS_API           => 'accounts',
+		WC_Payments_API_Client::CAPABILITIES_API       => 'accounts/capabilities',
```

Comment:
Since `key` and `value` are the same, I don't think that brings much value if we use an associate array here.
```suggestion
		WC_Payments_API_Client::ACCOUNTS_API,
		WC_Payments_API_Client::CAPABILITIES_API,
```

Reply:
This is like a micro optimization, but since the array is small, we could easily convert it to array.

Reply:
@zmaglica - you mentioned this https://gist.github.com/alcaeus/536156663fac96744eba77b3e133e50a in DM so I agree that makes a lot sense.

Since we’re using this quite often, this optimization is really worth doing. If so, instead of repeating the same thing for key and value, maybe we just switch all values into a fixed value such as `true`.

```suggestion
		WC_Payments_API_Client::ACCOUNTS_API           => true,
		WC_Payments_API_Client::CAPABILITIES_API       => true,
```

Reply:
I would rather keep in case we decide to refactor it and move from the constant from client class.